multiscene/js/switch-scene.js

```diff
@@ -0,0 +1,78 @@
+import {Component, Property} from "@wonderlandengine/api";
+
+/**
+ * switch-scene
+ */
+export class SwitchScene extends Component {
+	static TypeName = "switch-scene";
+	/* Properties that are configurable in the editor */
+	static Properties = {
+		target: Property.object(),
+		translationDuration: Property.float(0.5),
+		initialTranslate: Property.float(30),
+		portalRotateSpeed: Property.float(1.0),
+	};
+
+	startBool = false;
+	startUpdate = true;
+
+	onActivate() {
+		this.object.translateObject([0, 0, this.initialTranslate]);
+		console.error("activate got called on", this.scene.filename);
+		this.startUpdate = true;
+	}
+
+	onDeactivate() {
+		console.error("deactivate got called on", this.scene.filename);
+	}
+
+	update(dt) {
+		this.target.rotateAxisAngleDegObject(
+			[0, 0, 1],
+			dt * 10000 * this.portalRotateSpeed,
+		);
+		if (!this.startUpdate) return;
```

Comment:
Is that required? `update` can't be called before start normally

Reply:
Is this required because of the current bug with `onDeactivate`?

Reply:
yes true , its not required , will remove !